Makefile

```diff
@@ -356,6 +356,14 @@ e2e-gitlab: ## Runs e2e tests for GitLab only. TOKEN_TYPE is not used (since the
 e2e-gitlab: build-scorecard | $(GINKGO)
 	TEST_GITLAB_EXTERNAL=1 TOKEN_TYPE="PAT" $(GINKGO)  --race -p -vv -coverprofile=e2e-coverage.out --keep-separate-coverprofiles --focus ".*GitLab" ./...
 
+e2e-azure-devops-token: ## Runs e2e tests that require a AZURE_DEVOPS_AUTH_TOKEN
+e2e-azure-devops-token: build-scorecard check-env-gitlab | $(GINKGO)
+	TEST_AZURE_DEVOPS_EXTERNAL=1 TOKEN_TYPE="GITLAB_PAT" $(GINKGO) --race -p -vv -coverprofile=e2e-coverage.out --keep-separate-coverprofiles --focus '.*Azure DevOps' ./...
+
+e2e-azure-devops: ## Runs e2e tests for Azure DevOps only. TOKEN_TYPE is not used (since these are public APIs), but must be set to something
+e2e-azure-devops: build-scorecard | $(GINKGO)
+	TEST_AZURE_DEVOPS_EXTERNAL=1 TOKEN_TYPE="PAT" $(GINKGO)  --race -p -vv -coverprofile=e2e-coverage.out --keep-separate-coverprofiles --focus ".*Azure DevOps" ./...
+
```

Comment:
Does azdo have public APIs? all of my tokenless attempts in one of your earlier PRs returned access errors.

We could probably simplify this to just the `e2e-azure-devops` target

Reply:
It doesn't have any public APIs that are useful for Scorecard. I'll remove the `e2e-azure-devops` target.